modules/schema/src/main/java/org/apache/ignite/internal/schema/BinaryTupleSchema.java

```diff
@@ -289,10 +289,23 @@ public static int nullMapSize(int numElements) {
     }
 
     /**
-     * Returns the null map size in bytes if there are nullable elements, zero otherwise.
+     * Get offset of the byte that contains null-bit of a given tuple element.
+     *
+     * @param index Tuple element index.
+     * @return Offset of the required byte relative to the tuple start.
+     */
+    public static int getNullOffset(int index) {
+        return HEADER_SIZE + index / 8;
+    }
+
+    /**
+     * Get a null-bit mask corresponding to a given tuple element.
+     *
+     * @param index Tuple element index.
+     * @return Mask to extract the required null-bit.
      */
-    public int nullMapSize() {
-        return hasNullableElements() ? nullMapSize(elementCount()) : 0;
+    public static byte getNullMask(int index) {
```

Comment:
as far as I can see, vast majority of the methods within this class omit the `get` prefix. Lets follow this rule to make code more consistent

Reply:
Ok.